torchtune/models/clip/_position_embeddings.py

```diff
@@ -159,6 +164,13 @@ def _load_state_dict_hook(
                 tgt_patch_grid_size=int(math.sqrt(tgt_n_tokens_per_tile - 1)),
             )
 
+            if isinstance(inpt_local_pos_embed, DTensor):
```

Comment:
inpt_local_pos_embed is not a DTensor, because its full now

need to chech if self.local_token_positional_embedding is DTensor. We should probably do the same in the previous check (there you check if inpt_local_pos_embed is DTensor)